docs/ref/contrib/auth.txt

```diff
@@ -221,7 +221,8 @@ Methods
         Returns ``True`` if the user has the specified permission, where perm
         is in the format ``"<app label>.<permission codename>"``. (see
         documentation on :ref:`permissions <topic-authorization>`). If the user is
-        inactive, this method will always return ``False``.
+        inactive, this method will always return ``False``. For an active
+        superuser, this method will always return ``True``.
```

Comment:
Looks good, IMO we should add the same note to the `has_perms()` and `has_module_perms()`, also this behavior is already described in [`models.PermissionsMixin.has_perm()`](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.PermissionsMixin.has_perm), [`models.PermissionsMixin.has_perms()`](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.PermissionsMixin.has_perms), and [`models.PermissionsMixin.has_module_perms()`](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.PermissionsMixin.has_module_perms).

Reply:
Updated.